plenum/server/request_handlers/handler_interfaces/write_request_handler.py

```diff
@@ -43,16 +43,22 @@ def apply_request(self, request: Request, batch_ts):
     def revert_request(self, request: Request, batch_ts):
         pass
 
-    def update_state(self, txns, isCommitted=False):
+    def update_state(self, txns, is_committed=False):
         """
         Updates current state with a number of committed or
         not committed transactions
         """
+        for txn in txns:
+            self._update_state_with_single_txn(txn, is_committed=is_committed)
+
+    @abstractmethod
+    def _update_state_with_single_txn(self, txn, is_committed=True):
```

Comment:
Should `is_committed` be `False` here by default?

Reply:
yes, fixed